README.md

```diff
@@ -1 +1 @@
-testing semantic versioning 4.0.0
+testing semantic versioning 4.1.0
```

Comment:
- Consider adding a brief description of the changes made in the PR title or description for better clarity.